src/components/Dashboard/TalkDashboard.vue

```diff
@@ -541,7 +539,7 @@ function scrollEventCards({ direction }: { direction: 'backward' | 'forward' })
 	.talk-dashboard__actions {
 		:deep(.button-vue),
 		:deep(.v-popper--theme-dropdown) {
```

Comment:
```suggestion
		& > div {
```
I do not see such class in DOM